### ISSUE TITLE
'm' cannot be used as a value because it was exported using 'export type'.

### ISSUE THREAD

**Issue by joselevelsup** [aidenybai/million#71]
**Describe the bug**
As the title states

**To Reproduce**
```typescript
import { m, createElement } from "million";

const el = createElement(m("div", {}, ["Hello world"]));
```

**Expected behavior**
Everything compiles as it should.

**Screenshots**
![image](https://user-images.githubusercontent.com/10066172/125681308-d37a7c88-0f83-448b-832c-d9d6495154ba.png)

**Additional Context**
It looks like in the million.d.ts file, m, createElement, and patch are being exported as types and not as functions

**Comment by joselevelsup** [comment]
I have to revert to 0.2.2 in order for everything to work as is.

**Comment by aidenybai** [comment]
Seems like an issue with build process and how it's generating the d.ts file, looking into it

**Comment by aidenybai** [comment]
Fixed in v0.2.5
